### PR TITLE
Update VictoriaMetrics v1.100.1

### PR DESCRIPTION
**Pull request by takara9**
This PR is the part of the regular update

### CODE REVIEW

File: victoriametrics/Dockerfile [hunk 1]
```diff
@@ -33,7 +33,7 @@ COPY --from=build-single /go/src/github.com/VictoriaMetrics/VictoriaMetrics/bin/
 COPY --from=build-single /go/src/github.com/VictoriaMetrics/VictoriaMetrics/LICENSE /
 
 USER 10000:10000
-EXPOSE 8428
+EXPOSE 9428
```
Comment (umezawatakeshi) [inline victoriametrics/Dockerfile:36]: Why modified?

**Review by umezawatakeshi — CHANGES_REQUESTED** [review]

File: victoriametrics/Dockerfile [hunk 2]
```diff
@@ -33,7 +33,7 @@ COPY --from=build-single /go/src/github.com/VictoriaMetrics/VictoriaMetrics/bin/
 COPY --from=build-single /go/src/github.com/VictoriaMetrics/VictoriaMetrics/LICENSE /
 
 USER 10000:10000
-EXPOSE 8428
```
Comment (zoetrope) [inline victoriametrics/Dockerfile:36]: Why did you change the exposed port?

**Review by zoetrope — CHANGES_REQUESTED** [review]

File: victoriametrics/Dockerfile (hunk 1 above)
Comment (takara9) [reply victoriametrics/Dockerfile:36]: It is from upstream changing.

File: victoriametrics/Dockerfile (hunk 2 above)
Comment (takara9) [reply victoriametrics/Dockerfile:36]: It is from upstream changing.

Comment (takara9) [reply victoriametrics/Dockerfile:36]: victoria-metrics
curl -f -s -L -o CURRNT_VER https://raw.githubusercontent.com/VictoriaMetrics/VictoriaMetrics/v1.98.0-cluster/app/victoria-metrics/deployment/Dockerfile
curl -f -s -L -o UPDATE_VER https://raw.githubusercontent.com/VictoriaMetrics/VictoriaMetrics/v1.100.1-cluster/app/victoria-metrics/deployment/Dockerfile
--- CURRNT_VER  2024-04-15 05:45:36.775548489 +0000
+++ UPDATE_VER  2024-04-15 05:45:37.067548365 +0000
@@ -1,7 +1,7 @@
 ARG base_image
 FROM $base_image
 
-EXPOSE 8428
+EXPOSE 9428

Comment (zoetrope) [reply victoriametrics/Dockerfile:36]: I could not open that URLs.

I think it is VictoriaLogs Dockerfile?
https://github.com/VictoriaMetrics/VictoriaMetrics/pull/6082

Comment (takara9) [reply victoriametrics/Dockerfile:36]: @umezawatakeshi @zoetrope 
Sorry, it is my mistake.
the pot changing is only victoria-logs.

curl -f -s -L -o CURRNT_VER https://raw.githubusercontent.com/VictoriaMetrics/VictoriaMetrics/v1.98.0-cluster/app/victoria-logs/deployment/Dockerfile
curl -f -s -L -o UPDATE_VER https://raw.githubusercontent.com/VictoriaMetrics/VictoriaMetrics/v1.100.1-cluster/app/victoria-logs/deployment/Dockerfile
--- CURRNT_VER  2024-04-15 05:45:36.775548489 +0000
+++ UPDATE_VER  2024-04-15 05:45:37.067548365 +0000
@@ -1,7 +1,7 @@
 ARG base_image
 FROM $base_image
 
-EXPOSE 8428
+EXPOSE 9428

I will update currect port no

**Comment by umezawatakeshi** [comment]
I confirmed that the requested change is applied.
Please squash commits and push it.

**Review by umezawatakeshi — APPROVED** [review]

**Review by zoetrope — APPROVED** [review]
